include/fmt/core.h

```diff
@@ -279,7 +279,7 @@
 
 // Enable minimal optimizations for more compact code in debug mode.
 FMT_GCC_PRAGMA("GCC push_options")
-#ifndef __OPTIMIZE__
+#if !defined(__OPTIMIZE__) && !defined(__NVCOMPILER)
```

Comment:
This check should probably be moved here: https://github.com/fmtlib/fmt/blob/812733cc963b2e1d96f6ad2cb2d441c6fe8e4a5b/include/fmt/core.h#L35

Reply:
I moved it so `FMT_GCC_VERSION` is 0 for NVHPC.